Develop an SQL schema for SOPs

```diff
@@ -1,29 +1,4351 @@
+
+
+--
+-- TOC entry 2816 (class 2606 OID 17658)
+-- Name: STUDY_SAMPLES STUDY_SAMPLES_Study_ID_fkey; Type: FK CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."STUDY_SAMPLES"
+    ADD CONSTRAINT "STUDY_SAMPLES_Study_ID_fkey" FOREIGN KEY ("Study_ID") REFERENCES public."STUDY"("Study_ID") NOT VALID;
+
+
+--
+-- TOC entry 2810 (class 2606 OID 17628)
+-- Name: STUDY STUDY_Study_Assays_fkey; Type: FK CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."STUDY"
+    ADD CONSTRAINT "STUDY_Study_Assays_fkey" FOREIGN KEY ("Study_Assays") REFERENCES public."ASSAY"("Assay_ID") NOT VALID;
+
+
+--
+-- TOC entry 2811 (class 2606 OID 17633)
+-- Name: STUDY STUDY_Study_Contact_fkey; Type: FK CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."STUDY"
+    ADD CONSTRAINT "STUDY_Study_Contact_fkey" FOREIGN KEY ("Study_Contact") REFERENCES public."CONTACTS"("Contact_Person_ID") NOT VALID;
+
+
+--
+-- TOC entry 2812 (class 2606 OID 17638)
+-- Name: STUDY STUDY_Study_Design_Descriptors_fkey; Type: FK CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."STUDY"
+    ADD CONSTRAINT "STUDY_Study_Design_Descriptors_fkey" FOREIGN KEY ("Study_Design_Descriptors") REFERENCES public."STUDY_DESIGN_DESCRIPTORS"("Design_Descriptors_ID") NOT VALID;
+
+
+--
+-- TOC entry 2813 (class 2606 OID 17643)
+-- Name: STUDY STUDY_Study_Factors_fkey; Type: FK CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."STUDY"
+    ADD CONSTRAINT "STUDY_Study_Factors_fkey" FOREIGN KEY ("Study_Factors") REFERENCES public."STUDY_FACTORS"("Study_Factors_ID") NOT VALID;
+
+
+--
+-- TOC entry 2814 (class 2606 OID 17648)
+-- Name: STUDY STUDY_Study_ID_fkey; Type: FK CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."STUDY"
+    ADD CONSTRAINT "STUDY_Study_ID_fkey" FOREIGN KEY ("Study_ID") REFERENCES public."PROTOCOLS"("Protocol_ID") NOT VALID;
+
+
+--
+-- TOC entry 2815 (class 2606 OID 17653)
+-- Name: STUDY STUDY_Study_Publications_fkey; Type: FK CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."STUDY"
+    ADD CONSTRAINT "STUDY_Study_Publications_fkey" FOREIGN KEY ("Study_Publications") REFERENCES public."PUBLICATIONS"("Publication_ID") NOT VALID;
+
+
+-- Completed on 2020-10-30 17:55:35
+
+--
+-- PostgreSQL database dump complete
+--
+
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 12.3
+-- Dumped by pg_dump version 12.3
+
+-- Started on 2020-10-30 17:54:25
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+DROP DATABASE "MzTab and TraML";
```

Comment:
As stated previously, let's change all of the individual databases to a single database called `BFAIROmics`

Reply:
done